mwtext/content_transformers/wikidata2words.py

```diff
@@ -0,0 +1,37 @@
+import json
+import random
+
+import mwbase
+
+from .content_transformer import ContentTransformer
+
+
+class Wikidata2Words(ContentTransformer):
+    @classmethod
+    def from_siteinfo(cls, siteinfo, *args, **kwargs):
+        return cls(*args, **kwargs)
+
+    def transform(self, content):
+        doc = json.loads(content)
+        entity = mwbase.Entity.from_json(doc)
+        claims_tuples = list(self._extract_property_values(entity))
+        random.shuffle(claims_tuples)
+        return claims_tuples
+
+    @staticmethod
+    def _extract_property_values(entity):
+        properties = list(entity.properties.keys())
+        for prop in properties:
+            value_found = False
+            for statement in entity.properties[prop]:
+                claim = statement.claim
```

Comment:
probably not super important for training word2vec-ey type things on streams of PID/QID pairs, but might be interesting in later PRs to look at restricting to "truthy" claims https://www.mediawiki.org/wiki/Wikibase/Indexing/RDF_Dump_Format#Truthy_statements